app/javascript/gobierto_data/webapp/components/SidebarDatasets.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div>
+    <div
+      v-for="(item, index) in listDatasets"
+      :key="index"
+      class="gobierto-data-sidebar-datasets"
+    >
+      <div class="gobierto-data-sidebar-datasets-links-container">
+        <i
+          :class="{'rotate-caret': toggle !== index }"
+          class="fas fa-caret-down gobierto-data-sidebar-icon"
+          @click="handleToggle(index)"
+        />
+        <a
+          :href="'/datos/' + item.attributes.slug"
+          class="gobierto-data-sidebar-datasets-name"
+          @click.prevent="nav(item.attributes.slug, item.attributes.name, index)"
+        >{{ item.attributes.name }}
+        </a>
+        <div
+          v-show="toggle === index"
+          class="gobierto-data-sidebar-datasets-container-columns"
+        >
+          <div v-if="showLess">
+            <template v-for="(column, i) in lessColumns">
+              <span
+                :key="i"
+                class="gobierto-data-sidebar-datasets-links-columns"
+              >
+                {{ column }}
+              </span>
+            </template>
+          </div>
+          <div v-else>
+            <span
+              v-for="(column, i) in item.attributes.columns"
+              :key="i"
+              class="gobierto-data-sidebar-datasets-links-columns"
+            >
+              {{ i }}
+            </span>
+          </div>
+          <div v-if="Object.keys(item.attributes.columns).length > showMaxKeys">
+            <template
+              v-if="showLess"
+            >
+              <span
+                class="gobierto-data-sidebar-datasets-links-columns-see-more"
+                @click="showLess = false"
+              >
+                {{ labelshowAll }}
+              </span>
+            </template>
+            <template v-else>
+              <span
+                class="gobierto-data-sidebar-datasets-links-columns-see-more"
+                @click="showLess = true"
+              >
+                {{ labelshowLess }}
+              </span>
+            </template>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import { store } from "./../../lib/store";
+export default {
+  name: "SidebarDatasets",
+  data() {
+    return {
+      labelSets: "",
+      labelQueries: "",
+      labelCategories: "",
+      labelSeeMore: "",
+      labelSeeLess: "",
```

Comment:
Did you rename `labelSeeMore` with `labelshowAll`, perhaps? Same for `labelSeeLess`

Keep consistency about the names, and remember: every variable you use in the template must be in de `data()` function. That provides them reactivity. Otherwise becomes a constant.
The same but the other way around: if isn't in the template, remove it from `data()`